### PR TITLE
fix authorize

### PR DESCRIPTION
**Pull request by quanglx-noovado**
## Related Tickets:
- [#33080](https://edu-redmine.sun-asterisk.vn/issues/33080)

## WHAT this PR does?
Sửa lỗi phân quyền cho job và user

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by quanglx-noovado** [comment]
ready

File: app/Policies/CompanyPolicy.php [hunk 1]
```diff
@@ -53,7 +53,11 @@ public function create(User $user)
      */
     public function update(User $user, Company $company)
     {
-        return $user->company->id === $company->id;
+        if($user->role_id===config('user.employer')){
```
Comment (huyenttm-0901) [inline app/Policies/CompanyPolicy.php]: convention

File: app/Policies/JobPolicy.php [hunk 2]
```diff
@@ -53,7 +53,11 @@ public function create(User $user)
      */
     public function update(User $user, Job $job)
     {
-        return $user->company->id === $job->company->id;
+        if($user->role_id===config('user.employer')){
```
Comment (huyenttm-0901) [inline app/Policies/JobPolicy.php]: tương tự

**Comment by quanglx-noovado** [comment]
ready

File: resources/css/user.css [hunk 3]
```diff
@@ -332,4 +336,4 @@ button.chatbtn i, button.createbtn i {
     justify-content: center;
     margin: 20px auto;
     background-color: lavender;
-}
+}
```
Comment (congnx-2423) [inline resources/css/user.css]: endline

**Comment by quanglx-noovado** [comment]
ready

File: app/Http/Controllers/JobController.php [hunk 4]
```diff
@@ -154,6 +157,8 @@ public function apply($id)
         $user = Auth::user();
         $job = Job::findOrFail($id);
         $job->users()->attach($user->id, ['status' => config('job_config.waiting')]);
+        alert('Title Lorem Lorem Lorem', 'success');
```
Comment (huyenttm-0901) [inline app/Http/Controllers/JobController.php]: i18n

**Comment by quanglx-noovado** [comment]
ready

**Pull request merged**